conn.open（）时  碰到ORA-06413的问题

使用了很多网上的方法都没有效果

最后通过把path中oracle的路径删掉成功解决
